core/src/main/scala/com/salesforce/op/stages/impl/feature/SmartTextVectorizer.scala

```diff
@@ -81,23 +81,27 @@ class SmartTextVectorizer[T <: Text](uid: String = UID[SmartTextVectorizer[T]])(
     require(!dataset.isEmpty, "Input dataset cannot be empty")
 
     val maxCard = $(maxCardinality)
+    val minLenStdDev = $(minLengthStdDev)
     val shouldCleanText = $(cleanText)
 
     implicit val testStatsMonoid: Semigroup[TextStats] = TextStats.monoid(maxCard)
     val valueStats: Dataset[Array[TextStats]] = dataset.map(_.map(computeTextStats(_, shouldCleanText)).toArray)
     val aggregatedStats: Array[TextStats] = valueStats.reduce(_ + _)
 
-    val (isCategorical, topValues) = aggregatedStats.map { stats =>
+    val (isCategorical, isIgnorable, topValues) = aggregatedStats.map { stats =>
```

Comment:
can something be both ignorable and categorical? if not perhaps we should replace with an enum rather than multiplying our booleans...

Reply:
Not currently. Right now "ignorable" refers just to fields that would have been hashed otherwise. If a field is already low cardinality, I don't think we'd have a reason to ignore it in general, even if it was an ID or something.

We could make this into an enum with three choices (Pivot, Hash, Ignore) if you think that's clearer.

Reply:
Replaced! Mostly - I still need to propagate the changes to SmartTextMapVectorizer, but this did make things more readable. Good suggestion!